dtu-hpc-python3/setup-python3.sh

```diff
@@ -246,6 +246,10 @@ export PYTHON_BIN_PATH=`which python3`
 export TF_NEED_GCP=0
 export TF_NEED_HDFS=0
 export TF_NEED_CUDA=1
+export TF_NEED_JEMALLOC=0
+export TF_NEED_GCP=0
```

Comment:
both `TF_NEED_GCP` and `TF_NEED_HDFS` are repeated. Why is `TF_NEED_OPENCL=0` not required?